test/device.test.mjs

```diff
@@ -93,210 +94,210 @@ test("device: ViceBackend unsupported operations", async (t) => {
     );
   });
 
-  await t.test("reset throws unsupported", async () => {
+  await sub(async () => {
```

Comment:
@cursoragent There used to be nice strings with test descriptions. Now it is hard to see at a glance what each test does. Is there not a better way